<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltersPopup</class>
 <widget class="QWidget" name="FiltersPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="appStateGroup">
     <property name="title">
      <string>App State</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="appState_notInstalled">
        <property name="text">
         <string>Not Installed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appState_installed">
        <property name="text">
         <string>Installed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appState_updateRequired">
        <property name="text">
         <string>Update Required</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appState_updating">
        <property name="text">
         <string>Updating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appState_running">
        <property name="text">
         <string>Running</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="appTypeGroup">
     <property name="title">
      <string>App Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="appType_game">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appType_application">
        <property name="text">
         <string>Application</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="appType_music">
        <property name="text">
         <string>Music</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
